fearless/Common/Model/ChainRegistry/ChainModel.swift

```diff
@@ -1,6 +1,12 @@
 import Foundation
 import RobinHood
 
+extension ChainModel.Id {
+    var isOrml: Bool {
+        self == "9af9a64e6e4da8e3073901c3ff0cc4c3aad9563786d89daf6ad820b6e14a0b8b"
```

Comment:
Consider moving to some global `Constants.swift`, and it's a candidate for remote config.json in utils